bench/bench.cpp

```diff
@@ -320,7 +320,7 @@ class boost_pool_impl : public any_impl
         parser p;
         while(repeat--)
         {
-            monotonic_resource mr;
+            monotonic_resource mr(1024 * 64);
```

Comment:
Does this make a difference?

Reply:
I think this should be in its own commit because it affects performance, and it is currently buried in a larger optimization commit.

Reply:
Copy

Reply:
The difference made is negligible -- I added this to make our benchmark more similar to rapidjson